Apps/CocoaPods-FCM/src/View/SettingsView.swift

```diff
@@ -50,57 +52,52 @@ struct SettingsView: View {
                 ColorButton("Save") {
                     hideKeyboard() // makes all textfields lose focus so that @State variables are up-to-date with the textfield values.
 
-                    guard viewModel.settings.bqSecondsDelay > 0 else {
+                    guard viewModel.settings.flushInterval > 0 else {
                         alertMessage = "BQ seconds delay must be > 0"
                         return
                     }
 
-                    guard viewModel.settings.bqMinNumberTasks > 0 else {
+                    guard viewModel.settings.flushAt > 0 else {
                         alertMessage = "BQ min number tasks must be > 0"
                         return
                     }
 
-                    guard verifyTrackUrl() else {
+                    guard verifyHost(isCDN: true) else {
                         return
                     }
 
+                    nonBlockingMessage = "Settings saved. This will require an app restart to bring the changes in effect."
                     viewModel.saveSettings()
 
-                    // Re-initialize the SDK to make the config changes go into place immediately
-                    CustomerIO.initialize(siteId: viewModel.settings.siteId, apiKey: viewModel.settings.apiKey, region: .US) { config in
-                        viewModel.settings.configureCioSdk(config: &config)
-                    }
-
                     let didChangeSiteId = siteIdBeforeEditingSettings != viewModel.settings.siteId
                     if didChangeSiteId { // if siteid changed, we need to re-identify for the Customer.io SDK to get into a working state.
                         userManager.logout()
                     }
-
-                    done()
+                    timer.start(interval: TimeInterval(3)) {
+                        done()
+                    }
```

Comment:
what is it for?

Reply:
Added a [toast](https://github.com/customerio/customerio-ios/blob/9a6ea4430086c3344dab261dcfcf896c2a3bf347/Apps/CocoaPods-FCM/src/View/SettingsView.swift#L69) on settings screen to maintain consistency across all apps. The toast will disappear after 3 seconds and the user will be taken back to parent controller.